Monika After Story/game/script-topics.rpy

```diff
@@ -13478,3 +13478,64 @@ label monika_sweatercurse:
     m 1ekbsa "But I just want you to know that I'll always appreciate any project you put your heart into, [player]."
     m 1ekbfu "Whether you put a year or a day into something, I never want you to feel like your efforts are wasted."
     return
+    
+init 5 python:
+	addEvent(
+		Event(
+			persistent.event_database,
+			eventlabel='monika_caveallegory',
+			category=['philosophy'],
+			prompt="Seeing the real truth",
+			random=True
+			aff_range=(mas_aff.HAPPY, None)
```

Comment:
I see no reason to make this limited to happy+
```suggestion
```

Reply:
Mostly due to her expressions during this and her dialogue regarding her feelings towards the player, which I didn't think appropriate if the player had made her sad.

Reply:
I think you can make different flows in dialogue depending on the affection (if you ever want/need it). Since the topic itself doesn't have anything that she wouldn't discuss at low affection.